management/src/main/java/com/bitwisekaizen/sdss/dto/IscsiTargetSpec.java

```diff
@@ -0,0 +1,49 @@
+package com.bitwisekaizen.sdss.dto;
+
+import java.util.List;
+
+/**
+ * Spec that describes the ISCSI target
+ */
+public class IscsiTargetSpec {
```

Comment:
IscsiTargetSpec is also used in composition in IscsiTarget.  With your suggestion below, I'll rename this to IscsiTarget and the curren tIscsiTarget to something relating to LioBackedAgent.  Let's discuss offline though as there's also Nfs.  LioBackedIscsiAgent vs LioBackedNfsAgent?